tests/common/reboot.py

```diff
@@ -233,6 +236,10 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
     reboot_res, dut_datetime = perform_reboot(duthost, pool, reboot_command, reboot_helper, reboot_kwargs, reboot_type)
 
     wait_for_shutdown(duthost, localhost, delay, timeout, reboot_res)
```

Comment:
@xincunli-sonic why are we ignoring the reboot_res value? what is the guarantee the shutdown has completed?

Reply:
reboot_res is not ignoring, it consumes in wait_for_shutdown as the behavior no change. What I added is an event that notify the next step can proceed the further power on check if wait for shut down assertion finished.